app/src/main/java/ch/coredump/watertemp/activities/MapActivity.kt

```diff
@@ -77,6 +78,9 @@ class MapActivity : AppCompatActivity(), OnMapReadyCallback {
     // Views
     private var chart3days: LineChart? = null
 
+    // Activity indicator
+    private var progressCounter: ProgressCounter? = null
```

Comment:
You could use a [late init](https://kotlinlang.org/docs/reference/properties.html#late-initialized-properties-and-variables) variable for this use case, because you initialize it in `onCreate` and thus it is guaranteed that it is available every where else.